cmd/generate/config/rules/freemius.go

```diff
@@ -0,0 +1,44 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+	"regexp"
+)
+
+func Freemius() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "freemius-secret-key",
+		Description: "Detected a Freemius secret key, potentially exposing sensitive information.",
+		Regex:       regexp.MustCompile(`(?i)["']secret_key["']\s*=>\s*["'](sk_[\S]{29})["']`),
+		Keywords:    []string{"secret_key", "sk_"},
```

Comment:
only one of the keywords needs to match, and if I understand the samples, both are always present, so probably just "secret_key" would do?

Reply:
You're right, i removed the sk_ identifier